client/blocks/reader-post-card/featured-asset.jsx

```diff
@@ -2,37 +2,60 @@
  * External Dependencies
  */
 import React from 'react';
-import { head, filter, get } from 'lodash';
+import { find } from 'lodash';
 
 /**
  * Internal Dependencies
  */
 import FeaturedVideo from './featured-video';
 import FeaturedImage from './featured-image';
+import { isUrlLikelyAnImage } from '../../lib/post-normalizer/utils.js';
 
+function isCandidateForFeature( media ) {
+	if ( ! media ) {
+		return false;
+	}
+
+	if ( media.mediaType === 'image' ) {
+		const image = media;
+
+		const imageIsTallEnough = 350 <= image.width;
+		const imageIsWideEnough = 85 <= image.height;
+
+		return imageIsTallEnough && imageIsWideEnough;
+	} else if ( media.mediaType === 'video' ) {
+		// we need to have a thumbnail and know how to autoplay it
+		return media.thumbnailUrl && media.autoplayIframe;
+	}
+
+	return true;
+}
+
+/**
+ * Given a post:
+ *  1. prefer to return the post's featured image ( post.post_thumbnail )
+ *  2. if there is no usable featured image, use the media that appears first in the content of the post
+ *  3. if there is no eligible asset, return null
+ */
 const FeaturedAsset = ( { post } ) => {
 	if ( ! post ) {
 		return null;
 	}
 
-	const featuredImage = post.canonical_image;
-	let featuredAsset;
-
-	// only feature an embed if we know how to thumbnail & autoplay it
-	const featuredEmbed = head( filter( post.content_embeds, ( embed ) => embed.thumbnailUrl && embed.autoplayIframe ) );
+	// jetpack lies about thumbnails/featured_images so we need to make sure its actually an image
+	if ( post.featured_image && isUrlLikelyAnImage( post.featured_image ) ) {
```

Comment:
still need to check the feature to make sure it's big enough. Can use the dims on post_thumbnail I think?